container/values.yml

```diff
@@ -1,3 +1,5 @@
+start: /bin/bash /keg/keg-cli/containers/tap/run.sh
```

Comment:
This is not needed and should be removed.
A while ago, I started to migrate the sync actions into the `values.yml` file, but I never finished
Which is why you see this in other values files. I've since removed it, as seen [here](https://github.com/simpleviewinc/tap-events-force/blob/3bec8183f8404cd9f7e5f14e9474ae2dea0d6d0d/container/values.yml#L1) for the evf's image-from PR


I'd still like to do that migration at some point, but it not sure when.

Reply:
updated